src/components/Definition/Definition.module.scss

```diff
@@ -10,30 +10,87 @@
 }
 
 .definition_container {
-  position: absolute;
-  top: 50%;
-  left: 50%;
   width: 100%;
-  transform: translate(-50%, -50%);
   display: flex;
   justify-content: center;
-  align-items: center;
   opacity: 0;
-  animation: 1.6s ease-in 2s 1 fadeIn forwards;
+  animation: 1s ease-in 2.2s 1 fadeIn forwards;
+  margin-top: -32px;
 }
 
 .definition {
-  padding: clamp(2px, 0.5vw, 5px) clamp(5px, 1vw, 10px);
+  pointer-events: none;
+  display: flex;
+  align-items: center;
+  padding: 5px 12px;
   text-align: center;
   line-height: 1.5rem;
-  font-size: clamp(1rem, 2vw, 1.25rem);
-  background-color: $grey;
+  font-size: 1rem;
+  background-color: lighten($grey, 5%);
   border-radius: 10px;
 
-  span {
-    text-transform: uppercase;
-    letter-spacing: 1px;
-    font-weight: 700;
-    color: $blue;
+  .definition_sentence {
+    display: inline;
+    margin-left: 5px;
+  }
+}
+
+.logo {
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-size: 1rem;
```

Comment:
This reads as awful small to me, but I can see how it's more of an aesthetical decision than a technical one so I'll just point it out to you and live with whatever your think is best.